lib/Doctrine/DBAL/Schema/Expressions/Expression.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Doctrine\DBAL\Schema\Expressions;
+
+class Expression
+{
+    /** @var mixed $value */
+    protected $value;
+
+    /**
+     * @param mixed $value
+     */
+    public function __construct($value)
```

Comment:
Why would we want to accept `mixed` and cast it to a `string`? We should accept `string`.

Reply:
fixed